src/content/general/releases/index.md

```diff
@@ -87,13 +91,29 @@ Once we provide a new workload cluster release, be it major, minor, or patch, we
 
 Once deprecated, you can still continue to use the workload cluster release with existing clusters. In addition, as long as you have at least one cluster in your installation running the deprecated release, you can also create new clusters using that release. This allows you to create test clusters to test an upgrade from that release.
 
+### Pre-releases
+
+Pre-releases (when available) give you the option to test new features or use new functionality before the feature is released in a major version. At the same time helps the us release new features and test them in a realworld scenario with your help. When you test out a pre-release it helps find bugs or evaluate the impact of new functionality on the system. 
```

Comment:
We did do preview releases for only major ones until now but I wouldn't limit them just to majors. I can think of doing a minor preview release for an optional feature.

Reply:
Removed the limitation to major releases.